bridgeql/django/helpers.py

```diff
@@ -36,3 +41,30 @@ def __init__(self, content,
         else:
             data = json.dumps(content, indent=3, cls=encoder)
         HttpResponse.__init__(self, data, content_type, status)
+
+
+def get_local_apps():
+    if hasattr(settings, 'BASE_DIR'):
+        project_root = os.path.abspath(settings.BASE_DIR)
+    elif hasattr(settings, 'SITE_ROOT'):
+        project_root = os.path.abspath(settings.SITE_ROOT)
+    else:
+        project_root = None
+
+    if not project_root:
+        raise InvalidBridgeQLSettings(
+                'Could not find BASE_DIR and SITE_ROOT \
+                BRIDGEQL_ALLOWED_APPS needs to be initilised in settings')
+    _local_apps = []
+    for app in apps.get_app_configs():
+        if os.path.dirname(app.path) == project_root:
+            _local_apps.append(app.label)
+    return _local_apps
+
+
+def get_allowed_apps():
+    if hasattr(settings, 'BRIDGEQL_ALLOWED_APPS'):
```

Comment:
we can make use of `bridgeql_settings` here.
`bridgeql_settings.BRIDGEQL_ALLOWED_APPS` will return the settings value if it is present else it will return default value.

`from bridgeql.django.settings import bridgeql_settings`